backend/pennyme/pennycollector.py

```diff
@@ -99,7 +99,7 @@ def get_location_list_from_location_website(website) -> List[List[str]]:
         content = str(raw_locations[ind])
         if ind % 5 == 0:
             if ind > 0:
-                location_list.append(location)
+                location_list.append(content)
```

Comment:
was this a bug? How did this work before? :O

Reply:
I'm stupid because my change introduced a bug. I reverted the change now. The old code worked because it's a while loop so `location` is always initialized because `ind` is always 0 before it gets `>0` since it's incremented in every iteration. `ruff` was complaining about this,rightfully so because the code is ugly.

Reply:
That's what I though first. Then I refactored the code to remove the ruff complaint and realized that we are actually MISSING the last penny machine on every website! Crazy. 
I fixed the bug now, I assume there are some new machines incoming :)